draft-ietf-quic-transport.md

```diff
@@ -3791,9 +3791,9 @@ The two type bits from a Stream ID therefore identify streams as summarized in
 
 The first bi-directional stream opened by the client is stream 0.
 
-A QUIC endpoint MUST NOT reuse a Stream ID.  Streams can be used in any order.
-Streams that are used out of order result in opening all lower-numbered streams
-of the same type in the same direction.
+A QUIC endpoint MUST NOT reuse a Stream ID.  Streams of each type are created in
+numeric order.  Streams that are used out of order result in opening all
+lower-numbered streams of the same type in the same direction.
 
```

Comment:
It is a bit unclear what the difference is between "use", "create", "send", and "receive". I read such that "create" has to do with allocating a stream ID at or below the MAX ID but not necessarily sending. So a stream can be created silently and later some lower number stream can have its first frame sent - doing so accounts for a "use". For the receiver "use" either means "receive" or it means keep received frames buffered and pretend it is not there, but eventually "use" one.

I realize there is nothing enforceable about these distinctions, but it might affect buffering strategies.